### ISSUE TITLE
ISA Extensions

### ISSUE THREAD

**Issue by berkkis** [google/riscv-dv#92]
Hi, 
I have a question regarding ISA extension group G (IMFAD).
Will there be support for remainder ? 
If not what is the suggested practice for adding support for them ? 
Thank you,

**Comment by taoliug** [comment]
Yes, there's plan to support the FAD extension. Here's preferred the way to do it:
1) Create a development branch for each ISA extensions
2) File a bug for each ISA extensions
    - List all the features required to support this new ISA, collaborators can participate in the development by adding feature requirements, review PR, implement features etc.
3) Finish all development work under development branch before merging to the mainline. 

I will go ahead to create a development branch for A extension first. Regarding the complexity , I think A,D are relatively easy, may take a couple of weeks to implement. F extension will take more time, presumably a month or two.
